app/src/main/res/menu/drawer_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android" >
+    <group android:checkableBehavior="single">
+        <item
+            android:id="@+id/nav_camera"
+            android:title="View schedule" />
```

Comment:
Inconsistent capitalization between navigation items (capitalize the first letter of each word)

Reply:
Made others consistent